persistence/src/main/java/com/epam/esm/imp/GiftCertificateDaoImpl.java

```diff
@@ -0,0 +1,214 @@
+package com.epam.esm.imp;
```

Comment:
please use package corresponding to module. Module is persistence so you need com.epam.esm.service.impl

Reply:
Fixed